phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -330,161 +333,162 @@ protected PhoenixResultSet executeQuery(final CompilableStatement stmt, final Qu
-                            final long startTime = EnvironmentEdgeManager.currentTimeMillis();
-                            boolean success = false;
-                            boolean pointLookup = false;
-                            String tableName = null;
-                            PhoenixResultSet rs = null;
-                            try {
-                                PhoenixConnection conn = getConnection();
-                                conn.checkOpen();
-
-                                if (conn.getQueryServices().isUpgradeRequired() && !conn
-                                        .isRunningUpgrade()
-                                        && stmt.getOperation() != Operation.UPGRADE) {
-                                    throw new UpgradeRequiredException();
-                                }
-                                QueryPlan
-                                        plan =
-                                        stmt.compilePlan(PhoenixStatement.this,
-                                                Sequence.ValueOp.VALIDATE_SEQUENCE);
-                                // Send mutations to hbase, so they are visible to subsequent reads.
-                                // Use original plan for data table so that data and immutable indexes will be sent
-                                // TODO: for joins, we need to iterate through all tables, but we need the original table,
-                                // not the projected table, so plan.getContext().getResolver().getTables() won't work.
-                                if (plan.getTableRef() != null
-                                        && plan.getTableRef().getTable() != null && !Strings
-                                        .isNullOrEmpty(
-                                                plan.getTableRef().getTable().getPhysicalName()
-                                                        .toString())) {
-                                    tableName = plan.getTableRef().getTable().getPhysicalName()
-                                            .toString();
-                                }
-                                if (plan.getContext().getScanRanges().isPointLookup()) {
-                                    pointLookup = true;
-                                }
-                                Iterator<TableRef> tableRefs = plan.getSourceRefs().iterator();
-                                connection.getMutationState().sendUncommitted(tableRefs);
-                                plan =
-                                        connection.getQueryServices().getOptimizer()
-                                                .optimize(PhoenixStatement.this, plan);
-                                // this will create its own trace internally, so we don't wrap this
-                                // whole thing in tracing
-                                ResultIterator resultIterator = plan.iterator();
+            return CallRunner.run(
```

Comment:
I've checked this part of code locally because the here on the website it is really hard to review reformatted code.
As I see there is only a minor change in this part that setLastResultSet is not called.
To be honest I would not do this reformatting because it changes the git log/blame, makes it hard to search.

Reply:
It's the usual fight with the usless 120 character line width limit.
Since the code was already not well formatted, I may as well revert it.

Reply:
Addressed both concerns.